tests/test_template_init.py

```diff
@@ -110,6 +111,27 @@ def init_git(path: Path):
     repo.index.commit("chore: initialize template")
 
 
+def run_command(command: str, cwd: Path, **kwargs) -> None:
```

Comment:
Thank you for this. I was having a difficult time understanding the test output. I am usually slow with understanding new test suites, but this one was confusing because of all of the options and output it was providing.

Reply:
yeah in general i am not a fan of running tests with `-vv -s` (verbose mode, show stdout), but i am not the boss of this package. curious what you mean by options, i think there is only one nonstandard one, `--reuse-envs`, which causes the tests to not clean up hatch envs created by the generated packages (not the hatch envs of `pyos-package-template` itself, but the hatch envs of the temporary packages generated during testing)